rllib/core/models/tests/test_cnn_encoders.py

```diff
@@ -0,0 +1,129 @@
+import unittest
```

Comment:
Mostly the exact same file as before, but now unified between tf and torch.

Reply:
Thanks!